src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -80,6 +80,16 @@ namespace OpenMS
       return db_;
     }
 
+    inline static ModificationsDB * getCustomInstance(OpenMS::String unimod_file, OpenMS::String psimod_file)
```

Comment:
I don't think this is the correct way to do it because calling getCustomInstance(...) after getInstance was called at least once will just return the default one.